configure.ac

```diff
@@ -6,6 +6,18 @@ AC_INIT([libgnustep-corebase], [0.2], [bug-gnustep@gnu.org])
 AC_CONFIG_SRCDIR([Source/CFBase.c])
 AC_CONFIG_HEADERS([Source/config.h])
 
+if test -n "$GNUSTEP_MAKEFILES"; then
```

Comment:
Please use AS_IF() function instead of shell if-statement.

Reply:
I tried that, but I am getting the following warning when running `autoconf`. Do you have any suggestion how to fix that? I couldn’t figure it out from the link provided and searching Google...

```
configure.ac:9: warning: AC_REQUIRE: `AC_CONFIG_AUX_DIR_DEFAULT' was expanded before it was required
configure.ac:9: http://www.gnu.org/software/autoconf/manual/autoconf.html#Expanded-Before-Required
../../lib/m4sugar/m4sh.m4:639: AS_IF is expanded from...
configure.ac:9: the top level
```